core/krylovLogger.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <sstream>
+#include <iomanip>
+
+#include <boost/log/core.hpp>
+#include <boost/log/trivial.hpp>
+#include <boost/log/expressions.hpp>
+
+
+/**
+* Logger used to print warnings, errors, etc.
+*/
+class krylovLogger
+{
+public:
+	enum loggingLevel { DEBUG, INFO, WARNING, ERROR, FATAL };
+
+	krylovLogger() = default;
+	krylovLogger(loggingLevel level);
+	void set_loggingLevel(loggingLevel level);
+	void log_message(loggingLevel level, const std::string& msg);
+	static std::string stringStreamWrapperd(double val);
+
+private:
+	bool logToFile;
```

Comment:
It is not used at the moment. I started but then postponed the possibility to log to a file instead of terminal output. It is not much work, but I didn't want to delay the new TimeEvolver release for that. I removed the variable (and its initialization) for now. 
c4f05ab6